internal/env/map.go

```diff
@@ -0,0 +1,320 @@
+/* This file was generated using gen.pl and go fmt. */
+
+// dict is a helper function that allow one to easily get concreate values out of a map[string]interface{}
+package env
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/go-spatial/tegola/internal/dict"
+)
+
+type Map map[string]interface{}
```

Comment:
Should be changed to `env.Dict`. `map` is inconsistent with the other similar package and I think the word "map" could become ambiguous in the presence of geographical maps.